rest/middleware.go

```diff
@@ -14,6 +14,25 @@ import (
 	"github.com/pkg/errors"
 )
 
+type certDepotCheckMiddleware struct {
+	depotDisabled bool
+}
+
+// NewCertDepotCheckMiddleware returns an implementation of gimlet.Middleware
+// that returns an error if the internal cert depot is disabled.
+func NewCertDepotCheckMiddleware(disabled bool) *certDepotCheckMiddleware {
+	return &certDepotCheckMiddleware{depotDisabled: disabled}
+}
+
+func (m *certDepotCheckMiddleware) ServeHTTP(rw http.ResponseWriter, r *http.Request, next http.HandlerFunc) {
+	if m.depotDisabled {
+		gimlet.WriteResponse(rw, gimlet.MakeJSONErrorResponder(errors.New("certificate depot disabled")))
```

Comment:
Rather than writing a bad request response, can it return something like 403 forbidden?